Essentials/src/messages.properties

```
@@ -691,7 +691,15 @@ setworthCommandDescription=Set the sell value of an item.
 setworthCommandUsage=/<command> [itemname|id] <price>
 sheepMalformedColor=\u00a74Malformed color.
 shoutFormat=\u00a76[Shout]\u00a7r {0}
+editsignCommandClear=\u00a76Cleared your target sign.
+editsignCommandClearLine=\u00a76Cleared line\u00a7c {0}\u00a76 on your target sign.
```

```suggestion
editsignCommandClearLine=\u00a76Cleared line\u00a7c {0}\u00a76.
```

IMO it's redundant to specify `your target sign`; the player must be looking at the sign for this command to work in the first place.